.github/workflows/ci.yml

```diff
@@ -165,10 +165,22 @@ jobs:
           retention-days: 2
           if-no-files-found: error
 
+  # This job is only used to mark the CI as being done
+  #
+  # Allows us to add this as a required check in Github branch rules, as all the
+  # other jobs are subject to change
+  ci-done:
+    needs: [upload-coverage, linting, twemoji, pages]
+
+    runs-on: ubuntu-20.04
```

Comment:
is there anything lighter this could run on?

Reply:
Lightest that is available. https://github.com/actions/virtual-environments#available-environments

This will also be almost instant to spin up, as it is for all the other jobs. Difference is we dont setup anything (like python), which is what usually slows it down